src/styles/main.scss

```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import "./reset.scss";
+
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./utils/extends";
+@import "./blocks/page.scss";
+@import "./blocks/calendar.scss";
```

Comment:
It's better to use the underscore prefix for partial SCSS files to indicate that they are not to be compiled on their own. Rename the file to '_calendar.scss'.